util/schema-gen/src/org/javarosa/xform/schema/Harness.java

```diff
@@ -376,8 +382,14 @@ private static void processSchema(FormDef form) {
     private static FormDef loadFormDef(String[] args) {
         // Redirect output to syserr because sysout is being used for the
         // response, and must be kept clean.
-        PrintStream responseStream = System.out;
-        System.setOut(System.err);
+        PrintStream defaultOutStream = System.out;
+        PrintStream responseStream;
+        try {
+            responseStream = new PrintStream(System.err, false, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            responseStream = System.err;
+        }
+        System.setOut(responseStream);
```

Comment:
Weird, looks like there is some sort of strange github bug where in `File changes` view, the comment above (attached to line 392) shows in the wrong place...